### PR TITLE
Support for conditional logic following the ConditionalProof specification

### PR DESCRIPTION
**Pull request by theblockstalk**
Adds the ability to issue and verify VCs with multi-signatures and delegated signatures and combinations of the two, against DIDs that follow the ConditionalProof specification. See https://github.com/decentralized-identity/did-jwt/pull/272

https://github.com/w3c-ccg/verifiable-conditions

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -79,26 +79,58 @@ export {
...
+/**
+ * Adds an additional signature to an already signed VerifiableCredential 
+ * given a `jwt` and an `Issuer`.
+ *
+ * @param {JWT} jwt - The JWT to add a new signature to
+ * @param {Issuer} issuer - The issuer to use to sign the JWT
+ *
+ * @returns {Promise<JWT>} - The JWT with the new signature added
+ */
+export async function addSignatureToJwt(jwt: JWT, issuer: Issuer): Promise<JWT> {
```
Comment (mirceanis) [inline src/index.ts:131]: Please remove this since it's not implemented.

Comment (theblockstalk) [reply src/index.ts]: done

**Comment by mirceanis** [comment]
Looks like there's a benign failure in the linter.
Can you please run `yarn lint --fix` and push the results?

**Review by mirceanis — APPROVED** [review]
Looks great, thanks for adding this!

**Pull request merged**